tests/rptest/tests/throughput_limits_snc_test.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import random, time, math
+from enum import Enum
+from typing import Tuple
+from rptest.services.cluster import cluster
+from rptest.services.redpanda import MetricsEndpoint
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.rpk import RpkTool
+from ducktape.tests.test import TestContext
+
+# This file is about throughput limiting that works at shard/node/cluster (SNC)
+# levels, like cluster-wide and node-wide throughput limits
+
+
+class ThroughputLimitsSnc(RedpandaTest):
+    """
+    Throughput limiting that works at shard/node/cluster (SNC)
+    levels, like cluster-wide and node-wide throughput limits
+    """
+
+    # see later if need to split into more classes
+
+    def __init__(self, test_ctx: TestContext, *args, **kwargs):
+        super(ThroughputLimitsSnc, self).__init__(test_ctx,
+                                                  num_brokers=3,
+                                                  *args,
+                                                  **kwargs)
+        rnd_seed_override = test_ctx.globals.get("random_seed", True)
+        if rnd_seed_override is None:
```

Comment:
i think here you want to drop the `, True)` parameter to `get` ? for example, when you do not specify a random seed, i believe your seed with effectively be `random.Random(True)` which is probably not the intention?